editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -3491,8 +3491,8 @@ void ReflectionProbeGizmoPlugin::set_handle(const EditorNode3DGizmo *p_gizmo, in
 			d = 0.001;
 		}
 
-		extents[p_id] = d;
-		probe->set_extents(extents);
+		size[p_id] = d * 2;
```

Comment:
Note: May look weird at first but is needed.
The value `d` was set before as `extents`, and since we now use `size`, which is the `extents` * 2, we need to multiply by 2 here as well.
This also follows the same pattern as used here: https://github.com/godotengine/godot/pull/44183

I just wonder if we should multiply by 2 above, when assigning `d`. (`float d = ra[p_id] * 2`)
So the snapping and `0.001` is checked against the final `size` value. 
If the answer is yes, we need to change this on other locations as well.

Reply:
Might be worth adding a code comment if you think that line can be confusing.

Reply:
Agreed. Will wait for other opinions about the above described behaviour with `d` first.
Maybe some cleanup is needed in a follow up PR, in which case I would add a comment to all places (e.g. also the code places touched in https://github.com/godotengine/godot/pull/44183)

Reply:
Okay so this indeed does change behaviour: The snapping (holding `SHIFT`) is now done with a step of 2. Will change all locations now to restore this behaviour.